src/Admin/Views/ProviderOrganizations/AddExisting.cshtml

```diff
@@ -0,0 +1,98 @@
+@model ProviderOrganizationViewModel
+@using Bit.SharedWeb.Utilities
+
+@{
+    ViewData["Title"] = "Add Existing Organization";
+}
+
+<h1>Add Existing Organization</h1>
+<div class="row mb-2">
+    <div class="col">
+        <form class="form-inline mb-2" method="get">
+            @Html.HiddenFor(m => m.ProviderId)
+            <label class="sr-only" asp-for="OrganizationName">Organization Name</label>
+            <input type="text" class="form-control mb-2 mr-2 flex-fill" placeholder="Organization Name" asp-for="OrganizationName" name="name">
+            <label class="sr-only" asp-for="OrganizationOwnerEmail">Owner email</label>
+            <input type="email" class="form-control mb-2 mr-2 flex-fill" placeholder="Owner email" asp-for="OrganizationOwnerEmail" name="ownerEmail">
```

Comment:
The labels should be able to use `DisplayName` attributes from the model.

Reply:
Good point, I've made that change.